### PR TITLE
docs: add issue templates

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

**Comment by mik-laj** [comment]
What do you think about using [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) instead of plain text?

**Comment by cpcloud** [comment]
Ah, nice. Yeah, that'd be better.

**Comment by cpcloud** [comment]
Merging, so that I can see what it looks like.

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      required: true
+  - type: textarea
+    id: backends
+    attributes:
+      label: Backend(s)
+      description: What backend(s) are you using, if any?
+      placeholder: "Ex: DuckDB"
+    validations:
+      required: false
+  - type: checkboxes
```
Comment (mik-laj) [inline .github/ISSUE_TEMPLATE/feature-request.yml:40]: Screenshot 2022-11-11 at 16 51 06
I like to add a question to the forms for a new feature whether the person is interested in contributing. If so, it means that this ticket may need to be prioritized so as not to discourage the person from contributing.

Comment (cpcloud) [reply .github/ISSUE_TEMPLATE/feature-request.yml:40]: Great idea, I'll push up a PR for that :)

**Pull request merged**